### PR TITLE
ascanrules: More example alerts

### PR DESCRIPTION
**Pull request by kingthorin**
## Overview
- Scan Rules > Added example alert handling.
- Unit Tests > Added tests for the example alerts and references.
- CHANGELOG > Added change note.

## Related Issues
- zaproxy/zaproxy#6119

## Checklist
- [na] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [x] Write tests
- [x] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

### CODE REVIEW

**Review by ricekot — APPROVED** [review]

**Comment by thc202** [comment]
This has conflicts.

**Comment by kingthorin** [comment]
Otherwise good?

**Comment by thc202** [comment]
Yes.

**Comment by kingthorin** [comment]
Deconflicted

File: addOns/ascanrules/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - Maintenance changes.
 - The Spring Actuator Scan Rule now includes example alert functionality for documentation generation purposes (Issue 6119).
+- The following scan rules now include example alert functionality for documentation generation purposes (Issue 6119).
```
Comment (thc202) [inline addOns/ascanrules/CHANGELOG.md:10]: Merge them?

Comment (kingthorin) [reply addOns/ascanrules/CHANGELOG.md:10]: Yup, sorry didn't even read 🤦‍♂️

**Comment by kingthorin** [comment]
Changelog entries combined

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**